frontend-nextjs/README.md

```
@@ -0,0 +1,104 @@
+  - 起動時に`pnpm install`を実行して、ライブラリを再導入
+
+- VSCode拡張機能の設定
+- `frontend-nextjs/.vscode/extensions.json`
+- `frontend-nextjs/.vscode/settings.json`
+
+- devcontainerの確認
+- VSCodeの左下><をクリック　コンテナで再度開く
+- `frontend-nextjs-dev-container`を選択
+- コンテナ内からも`pnpm run dev`を実行できる
+
+## Biomeの導入
```

Markdownlint

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)